test/OpenSpaceToolkit/Mathematics/CurveFitting/Interpolation/BarycentricRational.test.cpp

```diff
@@ -0,0 +1,108 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// @project        Open Space Toolkit ▸ Mathematics
+/// @file           OpenSpaceToolkit/Mathematics/Interpolation/BarycentricRational.test.cpp
+/// @author         Vishwa Shah <vishwa@loftorbital.com>
+/// @license        Apache License 2.0
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <OpenSpaceToolkit/Core/Containers/Array.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Table.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/CurveFitting/Interpolation/BarycentricRational.hpp>
+
+#include <Global.test.hpp>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+TEST (OpenSpaceToolkit_Mathematics_Interpolator_BarycentricRational, Constructor)
+{
+
+    using ostk::core::types::Real ;
+
+    using ostk::math::obj::VectorXd ;
+    using ostk::math::curvefitting::interp::BarycentricRational ;
+
+    VectorXd x(6) ;
+    x(0) = 0.0 ;
+    x(1) = 1.0 ;
+    x(2) = 2.0 ;
+    x(3) = 3.0 ;
+    x(4) = 4.0 ;
+    x(5) = 5.0 ;
+
+    VectorXd y(6) ;
+    y(0) = 0.0 ;
+    y(1) = 3.0 ;
+    y(2) = 5.0 ;
+    y(3) = 6.0 ;
+    y(4) = 9.0 ;
+    y(5) = 15.0 ;
```

Comment:
You can use the [comma initializer](https://eigen.tuxfamily.org/dox/group__TutorialAdvancedInitialization.html) for more a compact code:

```suggestion
    VectorXd x(6) ;
    x << 0.0, 1.0, 2.0, 3.0, 4.0, 5.0 ;

    VectorXd y(6) ;
    y << 0.0, 3.0, 5.0, 6.0, 9.0, 15.0 ;
```